pkg/config/config_parser.go

```diff
@@ -236,3 +241,72 @@ func decodeCloudQueryBlock(block *hcl.Block, ctx *hcl.EvalContext) (CloudQuery,
 	cq.Logger = &logging.GlobalConfig
 	return cq, gohcl.DecodeBody(block.Body, ctx, &cq)
 }
+
+func validateConnection(connection *Connection) diag.Diagnostics {
```

Comment:
add tests to this function

Reply:
This function already had some test in https://github.com/cloudquery/cloudquery/pull/913/files#diff-23627973acbb422484ff2af98b8a1dcd8485d1703b280cae1b041867ac44f32fR9, and I added some more in https://github.com/cloudquery/cloudquery/pull/913/files#diff-23627973acbb422484ff2af98b8a1dcd8485d1703b280cae1b041867ac44f32fR52